tracer/build/_build/Build.Steps.cs

```diff
@@ -67,10 +71,7 @@ partial class Build
 
     AbsolutePath TempDirectory => (AbsolutePath)(IsWin ? Path.GetTempPath() : "/tmp/");
 
-    readonly string[] WafWindowsArchitectureFolders =
-    {
-        "win-x86", "win-x64"
-    };
+    readonly string[] WafWindowsArchitectureFolders = { "win-x86", "win-x64" };
```

Comment:
Mostly the changes to this file are reformatting?

Reply:
yes apart the added function to add an older waf file to the waf unit test's build directory , then ran into some trouble indenting it on Rider 😅